Utilities/get_titles_mismatches.py

```diff
@@ -0,0 +1,142 @@
+"""
+This script has functions to create Excel for mismatched titles, replace titles from
+JSON reference, and Excel with all the titles from Hindi JSON file
+"""
+import json
+import xlsxwriter
+
+
+def get_title_miss_matches(ref_json, check_json, xlsx_file, sheet_name="Sheet 1"):
+    """
+    This function create an excel sheet which contains all the miss-matched
```

Comment:
Product name first letter should be capital

Reply:
Spelling of mismatched is incorrect

Reply:
function 'creates'
mismatched what?

Reply:
mismatched titles, titles is in next title